src/main/java/de/muenchen/isi/configuration/SwaggerConfiguration.java

```diff
@@ -42,20 +42,7 @@ public OpenAPI openAPI() {
         final var authUrl = String.format("%s/realms/%s/protocol/openid-connect", this.authServer, this.realm);
 
         return new OpenAPI()
-                .components(new Components()
-                        .addSecuritySchemes("spring_oauth", new SecurityScheme()
-                                .type(SecurityScheme.Type.OAUTH2)
-                                .description("Oauth2 flow")
-                                .scheme("bearer")
-                                .bearerFormat("JWT")
-                                .flows(new OAuthFlows()
-                                        .password(new OAuthFlow()
-                                                .authorizationUrl(authUrl + "/auth")
-                                                .refreshUrl(authUrl + "/token")
-                                                .tokenUrl(authUrl + "/token")
-                                                .scopes(new Scopes()
-                                                        .addString("lhm_extended", "lhm_extended")))))
-                )
+                .components(new Components())
                 .security(Arrays.asList(
                         new SecurityRequirement().addList("spring_oauth")))
```

Comment:
Der Eintrag mit `spring_oauth` in Zeile 46 und 47 kann auch entfernt werden.

Dieser Eintrag schlägt sich in den API-Docs wie folgt nieder und hat keine Auswirkung.

![image](https://user-images.githubusercontent.com/56300801/211750607-54aa6df8-edc9-4116-badf-ba63b68aee1a.png)